cms/locale/cy/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,355 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2025-04-03 13:06+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=4; plural=(n==1) ? 0 : (n==2) ? 1 : (n != 8 && n != "
+"11) ? 2 : 3;\n"
+
+#: cms/articles/models.py:94
+#: cms/jinja2/templates/components/featured/featured-article.html:11
+msgid "Article"
+msgstr ""
+
+#: cms/articles/models.py:235
+msgid "Cite this article"
+msgstr ""
+
+#: cms/articles/models.py:237
+#: cms/jinja2/templates/components/contact_details/contact_details.html:5
+#: cms/methodology/models.py:147 cms/release_calendar/models.py:188
+msgid "Contact details"
+msgstr ""
+
+#: cms/core/blocks/related.py:25
+msgid "Related links"
+msgstr ""
+
+#: cms/core/utils.py:33
+msgid "Page"
+msgstr ""
+
+#: cms/core/utils.py:39
+#: cms/jinja2/templates/components/featured/featured-article.html:7
+msgid "Released"
+msgstr ""
+
+#: cms/jinja2/templates/base.html:49
+msgid "Beta"
+msgstr ""
+
+#: cms/jinja2/templates/base.html:50
+msgid "This is a new service."
+msgstr ""
+
+#: cms/jinja2/templates/base.html:51
+msgid "Menu"
+msgstr ""
+
+#: cms/jinja2/templates/base.html:52
+msgid "Main menu"
+msgstr ""
+
+#: cms/jinja2/templates/base.html:53
+msgid "Toggle main menu"
+msgstr ""
+
+#: cms/jinja2/templates/base.html:55
+msgid "All content is available under the"
+msgstr "Mae'r holl gynnwys ar gael o dan delerau'r"
```

Comment:
This was grabbed from the existing website for accuracy. Added so that tests can be conducted.